### PR TITLE
Finnish translations

### PR DESCRIPTION
**Pull request by saulipurhonen**
Added translations for Finnish language.

### CODE REVIEW

File: messages/index.js [hunk 1]
```diff
@@ -525,6 +544,7 @@ module.exports = {
     cs: 'musí vyhovět právě jednomu schématu v "oneOf"',
     de: 'muss genau einem der Schemata in "oneOf" entsprechen',
     es: 'debe coincidir con un solo esquema en "oneOf"',
+    fi: 'täytyy vastata "oneOf" -kohdassa määriteltyä skeemaa',
```
Comment (epoberezkin) [inline messages/index.js:547]: I think "exactly one" could be missing here?

File: messages/index.js [hunk 2]
```diff
@@ -549,6 +569,7 @@ module.exports = {
     cs: 'musí vyhovět regulárnímu výrazu "{{=e.params.pattern}}"',
     de: 'muss diesem Muster entsprechen: "{{=e.params.pattern}}"',
     es: 'debe coincidir con el patron "{{=e.params.pattern}}"',
+    fi: 'täytyy vastata mallia "{{=e.params.pattern}}"',
```
Comment (epoberezkin) [inline messages/index.js:572]: is there a better word for regular expression or is it what should be used?

File: messages/index.js [hunk 3]
```diff
@@ -573,6 +594,7 @@ module.exports = {
     cs: "musí obsahovat požadovanou položku {{=e.params.missingProperty}}",
     de: "muss das erforderliche Attribut {{=e.params.missingProperty}} enthalten",
     es: "debe tener la propiedad requerida {{=e.params.missingProperty}}",
+    fi: "{{=e.params.missingProperty}} vaaditaan",
```
Comment (epoberezkin) [inline messages/index.js:597]: please extend to match english message

File: messages/index.js [hunk 4]
```diff
@@ -698,6 +725,7 @@ module.exports = {
     cs: 'musí obsahovat položku vyhovující regulárnímu výrazu "{{=e.params.missingPattern}}"',
     de: 'muss ein Attribut nach folgendem Muster haben "{{=e.params.missingPattern}}"',
     es: 'la propiedad debe coincidir con el patrón "{{=e.params.missingPattern}}"',
+    fi: 'ominaisuuden on vastattava muotoa "{{=e.params.missingPattern}}"',
```
Comment (epoberezkin) [inline messages/index.js:728]: I don't think it's correct - it looks like "property mush match format" while the message should be "must have property matching pattern". Google suggests "kuvio" for pattern, although I am not sure this is a correct term for regular expression. In any case it should be the same as above.

File: messages/jtd.js [hunk 5]
```diff
@@ -175,6 +182,7 @@ module.exports = {
     cs: "musí být rovno jedné hodnotě z výčtu",
     de: "muss einem der vorgegebenen Werte entsprechen",
     es: "deber ser igual a uno de los valores predefinidos",
+    fi: "täytyy olla yhtä kuin jokin sallituista arvoista",
```
Comment (epoberezkin) [inline messages/jtd.js:185]: probably worth making the message in "enum" above the same?

**Review by epoberezkin — CHANGES_REQUESTED** [review]
Great - thank you very much! I've left comments/questions about some translations - I am probably wrong most of the time :) Also - could you please amend readme to add you as the author of Finnish messages?

Thank you!

File: messages/index.js [hunk 6]
```diff
@@ -128,6 +131,7 @@ module.exports = {
     cs: "{{#def.n}}nemůže mít víc, než {{=n}} prv{{#def.mPlural}}",
     de: "{{#def.n}}darf nicht mehr als {{=n}} Element{{#def.mPlural}} enthalten",
     es: "{{#def.n}}no debe tener más de {{=n}} elemento{{#def.mPlural}}",
+    fi: "{{#def.n}}saa sisältää enintään {{=n}}{{? n==1 }}:n elementin{{??}} elementtiä{{?}}",
```
[the remark this answers is not in the document]
Comment (saulipurhonen) [reply messages/index.js:134]: In this case where `n === 1` I'd write "saa sisältää yhden elementin" where "yhden" will be shortened to "1:n".

Other number counts would be " {n} elementtiä" and "1 elementti(ä)" would be wrong"

File: messages/index.js (hunk 2 above)
Comment (saulipurhonen) [reply messages/index.js:572]: I think "muotoa" would be better. It's basically same as "format" or "pattern" in this occasion.

File: messages/index.js [hunk 7]
```diff
@@ -106,6 +108,7 @@ module.exports = {
     cs: "schéma je false",
     de: "boolesches Schema ist falsch",
     es: "el esquema és falso",
+    fi: "boolean skeema on virheellinen",
```
[the remark this answers is not in the document]
Comment (saulipurhonen) [reply messages/index.js]: Changed this to stand for false value.

File: messages/index.js (hunk 1 above)
Comment (saulipurhonen) [reply messages/index.js]: Added statement for "exactly one".

File: messages/index.js (hunk 2 above)
Comment (saulipurhonen) [reply messages/index.js]: Changed for "muotoa".

File: messages/index.js (hunk 3 above)
Comment (saulipurhonen) [reply messages/index.js]: Extended to match original phrase.

File: messages/index.js (hunk 4 above)
Comment (saulipurhonen) [reply messages/index.js]: You're absolutely right. "Kuvio" would suit better for textures but in this case I'd go with "kaava" (in this case "kaavaa"), which would translate to "pattern / formula / format".

Changes made to this particular word and phrasing to match original string"

File: messages/jtd.js (hunk 5 above)
Comment (saulipurhonen) [reply messages/jtd.js:185]: Both "enum" properties should have identical translations now.

**Comment by epoberezkin** [comment]
Thank you!

**Pull request merged**